hushline/routes.py

```diff
@@ -118,7 +121,7 @@ def submit_message(username: str) -> Response | str:
 
         if not user:
             flash("🫥 User not found.")
-            return redirect(url_for("index"))
+            return redirect(url_for("index")), 404
```

Comment:
You cannot redirect using a 404. Either we have a dedicated 404 page and flash on that or we redirect to index and flash.

Reply:
👍